### DISCUSSION TITLE
[Buildings]

### DISCUSSION THREAD

**Discussion by bkaplan36** [OvertureMaps/data#239] [Discussion]
### Category Feedback

Regarding the metadata for buildings, does the update_time represent the time the data was updated in the Overturemaps database or the data it was updated in the source dataset (Microsoft, OSM, etc).   Does the metadata say anything about when the feature was extracted or the time of the image used for the extraction?  Does the record_id link back to the source which would mean we could use the record_id and find the original record in the source data?

**Comment by jenningsanderson** [comment]
Hi @bkaplan36 — Yes, this is exactly right.

`update_time` is available as a per-source attribute. In the case of buildings from OpenStreetMap, this is the last time that the building was updated in OSM. 

In the case of Microsoft buildings, `update_time` refers to the imagery date, if available.

`record_id` is the `id` from the original dataset, so you should be able to trace it back, if you'd like. Sometimes these are not unique (in the case of the ML data where they might be a form of a geohash).
